pages/account_lists/[accountListId]/accept_invite/[inviteId].page.tsx

```diff
@@ -0,0 +1,28 @@
+import { GetServerSideProps } from 'next';
+import { ReactNode } from 'react';
+import { getSession } from 'next-auth/react';
+
+// This page redirect old email invite links to the new page that handles invites
+const InvitePage = (): ReactNode => null;
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const session = await getSession(context);
+  const { accountListId, inviteId, code } = context.query;
+
+  const redirectURL =
+    accountListId && inviteId && code
+      ? `/acceptInvite?accountListId=${accountListId}&accountInviteId=${inviteId}&inviteCode=${code}`
+      : `/accountLists/${accountListId}`;
+
+  return {
+    redirect: {
+      destination: session ? redirectURL : '/login',
+      permanent: true,
+    },
```

Comment:
Login redirects should technically be `permanent: false`.

```suggestion
    redirect: session ? {
      destination: redirectURL,
      permanent: true,
    } : {
      destination: '/login',
      permanent: false,
    },
```